modules/module-mysql/dev/config/sync_rules.yaml

```diff
@@ -0,0 +1,12 @@
+# See Documentation for more information:
+# https://docs.powersync.com/usage/sync-rules
+# Note that changes to this file are not watched.
+# The service needs to be restarted for changes to take effect.
+
+# Note that specifying the schema is currently required due to the default
+# schema being specified as `public`, but in mysql the schema is the database name
```

Comment:
Is this still the case, with `defaultSchema` now being the database name?

Reply:
No you are right, no longer needed :)